### PR TITLE
Medaka 1.3.2 update

### PR DESCRIPTION
**Pull request by NatPRoach**
I doubt this will work on the first go because I didn't update the test-data at all and medaka switched around its models at some point. Wanted to make the PR while I work on updating tests though.
There are a number of medaka tools functions we do not currently support:
```
(medaka) nathan@nathan-MS-7C84:~/Documents/galaxyworks/tools-iuc/tools/medaka$ medaka tools --help
usage: medaka tools [-h] [--debug | --quiet]
                    {haploid2diploid,diploid2haploid,classify_variants,annotate,consensus2vcf,vcf2tsv,homozygous_regions,resolve_model,is_rle_model,get_alignment_params,rlebam,list_models,hdf_to_bed,download_models}
                    ...

optional arguments:
  -h, --help            show this help message and exit
  --debug               Verbose logging of debug information. (default: 20)
  --quiet               Minimal logging; warnings only). (default: 20)

tools:
  valid tool commands

  {haploid2diploid,diploid2haploid,classify_variants,annotate,consensus2vcf,vcf2tsv,homozygous_regions,resolve_model,is_rle_model,get_alignment_params,rlebam,list_models,hdf_to_bed,download_models}
                        additional help
    haploid2diploid     Merge two haploid VCFs into a diploid VCF.
    diploid2haploid     Split a diploid VCF into two haploid VCFs.
    classify_variants   Classify variants and write vcf for each type and subtype.
    annotate            Annotate vcf with read depth and supporting reads info fields.
    consensus2vcf       Call variants by alignment of a consensus sequence to a reference.
    vcf2tsv             convert vcf to tsv, unpacking info and sample columns.
    homozygous_regions  Find homozygous regions from a diploid vcf.
    resolve_model       Resolve model and print full file path.
    is_rle_model        Check if a model is an RLE model.
    get_alignment_params
                        Get alignment parameters appropriate for a model
    list_models         List all models.
    hdf_to_bed          Write a bed file of regions spanned by a hdf file.
    download_models     Download model files for any models not already installed.
```
as well as a number of medaka subcommands we don't currently support:
```
(medaka) nathan@nathan-MS-7C84:~/Documents/galaxyworks/tools-iuc/tools/medaka$ medaka --help 
usage: medaka [-h] [--version]
              {compress_bam,features,train,consensus,smolecule,consensus_from_features,fastrle,stitch,variant,snp,tools} ...

optional arguments:
  -h, --help            show this help message and exit
  --version             show program's version number and exit

subcommands:
  valid commands

  {compress_bam,features,train,consensus,smolecule,consensus_from_features,fastrle,stitch,variant,snp,tools}
                        additional help
    compress_bam        Compress an alignment into RLE form.
    features            Create features for inference.
    train               Train a model from features.
    consensus           Run inference from a trained model and alignments.
    smolecule           Create consensus sequences from single-molecule reads.
    consensus_from_features
                        Run inference from a trained model on existing features.
    fastrle             Create run-length encoded fastq (lengths in quality track).
    stitch              Stitch together output from medaka consensus into final output.
    variant             Decode probabilities to VCF.
    snp                 Decode probabilities to SNPs.
    tools               tools subcommand.

```

May be worth adding support for these in future, but not an immediate priority I don't think.

FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/medaka/convert_VCF_info_fields.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 from collections import OrderedDict
 from math import log10
 
-from scipy.stats import fisher_exact
+import scipy
```
Comment (bgruening) [inline tools/medaka/convert_VCF_info_fields.py:14]: but you are using scipy here? Do you know from where this gets pulled in?

Comment (NatPRoach) [reply tools/medaka/convert_VCF_info_fields.py:14]: I don't know where it gets pulled in but I can try to find out.

Comment (NatPRoach) [reply tools/medaka/convert_VCF_info_fields.py:14]: It appears to be a dependency of whatshap, which is a dependency of medaka

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Danke!

**Comment by NatPRoach** [comment]
Thank you!

**Pull request merged**